sync3/sort.go

```diff
@@ -145,10 +145,10 @@ func (s *SortableRooms) comparatorSortByName(i, j int) int {
 
 func (s *SortableRooms) comparatorSortByRecency(i, j int) int {
 	ri, rj := s.resolveRooms(i, j)
-	if ri.LastMessageTimestamp == rj.LastMessageTimestamp {
+	if ri.LastActivityTimestamp == rj.LastActivityTimestamp {
 		return 0
 	}
-	if ri.LastMessageTimestamp > rj.LastMessageTimestamp {
+	if ri.LastActivityTimestamp > rj.LastActivityTimestamp {
```

Comment:
I wonder if `LastActivityTimestamp` is too vague here. It implies things like typing/receipts can affect the timestamp as they are activity. I understand not wanting to call it `LastMessageTimestamp` anymore though. Perhaps `LastInterestedEventTimestamp` or something? Slightly mouthy but it clarifies that it is indeed an `origin_server_ts` of an event, by some definition of "interested" which implies some kind of filter, without going too specific and mentioning bump event types.

Reply:
Done in e665070.